about/about.html

```diff
@@ -0,0 +1,84 @@
+  <title>Barbershop KRS-Checkpoint1 about</title>
+</head>
+
+<body>
+  <nav class="nav" id="nav">
+    <ul>
+      <li><a class="active">SAFETY PROTOCOL</a></li>
+      <li class="dropdown"><a href="/landing-page/salon.html">WELCOME TO</a>
+        <ul class="dropdown-content">
+          <li><a href="/about/about.html">ABOUT</a></li>
+          <li><a>SERVICE POLICY</a></li>
+        </ul></li>
+      <li><a>#KRSGALMSQUAD</a></li>
+      <li><a href="/tips-trix/tips.html">TIPS & TRIX</a></li>
+      <li class="dropdown"><a>SERVICES</a>
+        <ul class="dropdown-content">
+          <li><a>HAIR</a></li>
+          <li><a>BRAIDS</a></li>
+          <li><a>LOCS</a></li>
+          <li><a>WAXING</a></li>
+          <li><a>EYELASH EXTENSIONS</a></li>
+          <li><a>BEAUTY</a></li>
+        </ul>
+      </li>
+      <li><a>BOOK NOW</a></li>
+      <li><a>BALENCED</a></li>
+      <li><a href="/contact/contact.html">CONTACT</a></li>
+      <li class="icon">
+        <a href="javascript:void(0);" onclick="myFunction()">
+          <i class="fa fa-bars"></i></a>
+      </li>
+    </ul>
+  </nav>
+    <script src="salon-script.js"></script>
+
+<main>
+  
+  <section class="block1">
+  <h1>WE ARE KRS.</h1>
+  <img src="/images/family2.jpg" alt="family photo" class="about-hero-img">
+  <div class="blk1-p">
+    <p>KRS Hair & Wax Studio is family owned and operated, established in 2005 by partners in marriage and business, Christopher and Kimberly Wilder. New guests frequently ask what KRS means.  We are pleased to share that it stands for Kimberly, Rai and Symonne;  Christopher's wife and daughters, because his philosophy is that 'family makes the man.' </p>
+    <p>KRS Hair & Wax Studio is an energetic and professional, full service salon, which specializes in hair care for all textures and ethnicities. We are known for our expertise in natural hair care, incredible hair extensions, precise cuts, creative color and luxurious lash extensions. Additionally, our Aesthetician is ranked by the WISN 12 A-List as one of the top two Waxers in the City of Milwaukee. With a focus on impeccable guest services and cultural diversity, we are Milwaukee's premier full service salon.</p>
+  </div>
+</section>
+
+  <section class="block2">
+    <p>We are pleased to offer our guests the very best salon exclusive products such as Paul Mitchell, Mizani, Keratin Complex, Enjoy, Skin Scripts, Relax and Wax, JPA Elevated CBD products, KRS Eyebrow Fairy brow cosmetics  and Butterscotch Baby Total Body Experience.  While we are thoroughly impressed with all of our product lines, we are especially proud to feature Butterscotch Baby TBE and JPA Elevated, two hand-crafted, organic hair, skin & body care lines, made locally by  female-owned companies. For more info, please visit their social media platforms at </p>
+    <a> https://instagram.com/jpaelevated?igshid=1gcmjcg0nakso  </a>
```

Comment:
these aren't links bc you need to put the link in the href="" attribuel <a href="link-goes-here" >Name of Link Goes here</a>